modules/oe_content_reference/oe_content_reference.info.yml

```diff
@@ -0,0 +1,13 @@
+name: OpenEuropa Content Reference Field
```

Comment:
Rename this module in `oe_content_reference_code_field`, so to avoid confusion with the other meaning of the word `reference` in Drupal.

Reply:
done